src/shared/strings.ts

```diff
@@ -50,6 +51,14 @@ export function diffInMinutes(from: Date | string, to: Date | string): number {
   return Math.floor((new Date(to).getTime() - new Date(from).getTime()) / 1000 / 60);
 }
 
+export function sortByDateDesc(contents: any[]) : void{
```

Comment:
change to `compareDate(d1: ..., d2: ...)`. do sorting in the call sites instead.